tests/fixtures/vue-component/component-with-v-bind-props.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div>
+    <ChildComponent v-bind="$props" bar="baz" />
+  </div>
+</template>
+
+<script>
+    export default {
```

Comment:
$props represents all properties, so I suggest to add at least one property to the component to make sure that one is added to the component as well?

Reply:
Yes that's right. 
But on the Twig side, the properties are automatically inherited from top to bottom.

If I now e.g. turning `$props` into `_props` I have two problems:
* _props can not loger use as variable.
* a Twig Block is neede. Like this: `<twig> {% set _props = [foo, bar] %} </twig>`

Reply:
I have created a solution which actively passes the props.

```php
                if ($value === '"$props"') {
                    foreach ($this->properties as $property) {
                        $variables[] = (clone $property)->setValue($property->getName());
                    }
                } else {
                    $this->vBind = $value;
                }
```